cmd/agent/api/agent/cpython_status.go

```diff
@@ -1,32 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-2019 Datadog, Inc.
-
-// Package agent implements the api endpoints for the `/agent` prefix.
-// This group of endpoints is meant to provide high-level functionalities
-// at the agent level.
-
-// +build cpython
-
-package agent
-
-import (
-	"encoding/json"
-	"net/http"
-
-	"github.com/DataDog/datadog-agent/pkg/collector/py"
-	"github.com/DataDog/datadog-agent/pkg/util/log"
-)
-
-func getPythonStatus(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
```

Comment:
Would be nice to add this code to the new `getPythonStatus` even if it's commented out, at least to know what the function was supposed to do.